lib/WP_Auth0_Lock10_Options.php

```diff
@@ -196,7 +196,7 @@ public function has_custom_signup_fields() {
   }
 
   public function get_sso_options() {
-    $options["scope"] = "openid ";
+    $options["scope"] = "openid email identities ";
```

Comment:
I don't remember if `identities` is a valid scope. Is it?

Reply:
@lbalmaceda - Used the code below as an example:

https://github.com/auth0/wp-auth0/blob/master/lib/WP_Auth0_Lock_Options.php#L227

During debugging, I dumped the $userinfo object and has identities so I think 👍

Reply:
This will only work in non OIDC conformance, now the plugin doesn't work in OIDC Conformant mode anyway. However, just want you to be aware. This can be refactored in 4.0.0 to use the MGMT API get user.

Reply:
True. Then maybe it's worth to document it somewhere in the readme letting people know that it won't work on legacy clients.